### PR TITLE
Update serverless feature to latest Knative version v0.13

### PR DESCRIPTION
**Pull request by Uvindu96**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: scenarios/scenario-14/hello-world-serverless.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+apiVersion: wso2.com/v1alpha1
```
Comment (pubudu538) [inline scenarios/scenario-14/hello-world-serverless.yaml]: Please add the license header.

File: scenarios/scenario-14/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+swagger: "2.0"
```
Comment (pubudu538) [inline scenarios/scenario-14/swagger.yaml]: License Header

File: scenarios/scenario-14/hello-world-serverless.yaml (hunk 1 above)
Comment (Uvindu96) [reply scenarios/scenario-14/hello-world-serverless.yaml]: license header added

File: scenarios/scenario-14/swagger.yaml (hunk 2 above)
Comment (Uvindu96) [reply scenarios/scenario-14/swagger.yaml]: license header added

**Pull request merged**